### ISSUE TITLE
Create fuzzjobs with "-" in their name causes a crash

### ISSUE THREAD

**Issue by TomSie** [siemens/fluffi#181]
Please modify the web gui to prevent "-" in fuzzjob names for now

**Comment by p0wer0xff** [comment]
The reason for this is that a database name is generated from the fuzzjob name.

Here a job named `fuzzjob-h2` was created:
```
fluffiweb_1  | 2020/05/13 13:51:45 [warn] 11#11: *2710 a client request body is buffered to a temporary file /var/cache/nginx/client_temp/0000000002, client: 172.18.0.1, server: , request: "POST /projects/createProject HTTP/1.0", host: "localhost:8880", referrer: "http://web.fluffi/projects/createProject"
fluffiweb_1  | ERROR 1064 (42000) at line 25: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '-h2' at line 1
fluffiweb_1  | Could not execute mysql command
fluffiweb_1  | Traceback (most recent call last):
[...]
```

This causes an entry in the `fluffi_gm` database for the fuzzjob, but does not actually create that database. Accessing the web interface after this causes the whole webapp to crash:
```
fluffiweb_1  | sqlalchemy.exc.InternalError: (pymysql.err.InternalError) (1049, "Unknown database 'fluffi_fuzzjob-h2'") (Background on this error at: http://sqlalche.me/e/2j85)
fluffiweb_1  | 2020/05/13 13:52:22 [error] 11#11: *2716 upstream prematurely closed connection while reading response header from upstream, client: 172.18.0.1, server: , request: "GET / HTTP/1.0", upstream: "uwsgi://unix:///tmp/uwsgi.sock:", host: "localhost:8880"
```

It should be enough to filter database name creation.

[edit:] Also, I'm pretty sure we have seen this before with @maverickwoo, but I cannot find any reference to it right now...

[edit2:] found it: we have it [in the docs](https://github.com/siemens/fluffi/blame/3312032ba2b3ec97f5c93e6c00c9596dd2e58917/usage.md#L96)

**Comment by jones718** [comment]
Fixed with pull request #192
